homeassistant/components/unifi/config_flow.py

```diff
@@ -341,6 +342,10 @@ async def async_step_device_tracker(self, user_input=None):
                         CONF_TRACK_DEVICES,
                         default=self.controller.option_track_devices,
                     ): bool,
+                    vol.Optional(
+                        CONF_DEVICE_UPDATES,
+                        default=self.controller.option_device_updates,
+                    ): bool,
```

Comment:
Why make this configurable?

Reply:
I agree, more of those options should be removed due to general improvements in home assistant